src/features/list-view/ListView.js

```diff
@@ -22,12 +22,27 @@ type Props = {
     getGridHeader: (columnIndex: number) => any,
     getGridHeaderSort?: (columnIndex: number) => typeof SORT_ORDER_ASCENDING | typeof SORT_ORDER_DESCENDING | null,
     height: number,
+    onCellHover?: ({|
```

Comment:
On second thought, I'd prefer that we move this to EUA. I couldn't think of a reason previously but am realizing that any work in BUI tends to carry undesirable overhead.